lib/pennyworth/commands/setup_command.rb

```diff
@@ -77,6 +78,26 @@ def reload_udev_rules
       Cheetah.run "sudo", "/sbin/udevadm", "trigger"
     end
 
+    def show_warning_for_unsupported_platforms
```

Comment:
Assignment Branch Condition size for show_warning_for_unsupported_platforms is too high. [16.58/15]